__tests__/utilities/test-utils.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs');
+const _ = require('lodash');
+const path = require("path");
+const RudderCDK = require("rudder-transformer-cdk");
+
+function formTestParams(dest, transformAt) {
+	const version = "v0";
+  
+  const transformer = require(
+    path.resolve(__dirname + `../../../${version}/destinations/${dest}/transform`)
+  );
+  
+  //for router test
+  let trCat = '';
+  if (transformAt === 'router') {
+    trCat += 'router_'
+  }
+  const inputDataFile = fs.readFileSync(
+    path.resolve(__dirname, `../data/${dest}_${trCat}input.json`)
+  );
+  const outputDataFile = fs.readFileSync(
+    path.resolve(__dirname, `../data/${dest}_${trCat}output.json`)
+  );
+  const inputData = JSON.parse(inputDataFile);
+  const expected = JSON.parse(outputDataFile);
+	const cdkDest = ['variance'].filter(({ name }) => name === dest)[0];
+	return {
+		input: inputData,
+		expected,
+		transformer,
+		iscdkDest: _.isEmpty(cdkDest)
+	};
+}
+
+function executeTransformationTest(dest, transformAt) {
+  const testParams = formTestParams(dest, transformAt);
+  const { iscdkDest, transformer, expected, input } = testParams;
+  
+  const basePath = path.resolve(__dirname, "../../cdk");
+  const factory = new RudderCDK.ConfigFactory(basePath);
+
+  describe(`${dest} Tests`, () => {
+    it(`${dest} - ${transformAt} tests`, async () => {
```

Comment:
Introduce a new `description` field in test case data and use it here.

Reply:
Is that not a big change for every destination ?

Reply:
We should do it nevertheless. Start with CDK destinations.